src/test/java/systemtests/CardFolderSystemTest.java

```diff
@@ -121,7 +130,9 @@ public ResultDisplayHandle getResultDisplay() {
      */
     protected void executeCommand(String command) {
         rememberStates();
-        // Injects a fixed clock before executing a command so that the time stamp shown in the status bar
+        // Injects a fixed cloc
+        //
+        // k before executing a command so that the time stamp shown in the status bar
```

Comment:
some weird indentation here

Reply:
Oh LOL the GUI tests moved the mouse and inserted a new line when I wasn't looking.